common/lib/api-client/auth.js

```diff
@@ -0,0 +1,58 @@
+const axios = require('axios')
+
+function Auth({ timeout = 10000, authUrl, username, password } = {}) {
```

Comment:
do we need to destructure here? Could you just pass `config` through?

Reply:
I quite like  destructuring when there's only a few options as it helps you see what's expected. It also makes it neater to use so that you don't need to do `options.{property}`.